packages/block-library/src/latest-comments/style.scss

```diff
@@ -1,3 +1,8 @@
+.wp-block-latest-comments {
+	padding-left: 0;
+	margin-left: 0;
```

Comment:
I tested with TT1 blocks and this change is not reflected in the `editor`. Also in front-end the `margin-left` is overridden with `auto` and if it wasn't it would place this block at the left of the page differently from other blocks that would retain `margin-left:auto`.

So maybe patch the editor and `padding-left` should be enough 🤔 . I'm not sure how other themes behave to this though..

Reply:
Right, I should not have tested with Twenty Twenty-One 🤦

Reply:
Thank you for the PR!

The padding change is great, but the margin change, I believe, will cause headaches. Many themes, to Nik's point, center blocks using margin-left: auto, and by setting it to zero, you might make the latest comments block bump againt the left edge. 

We have the same challenge with the Image block, which is wrapped in a `figure` element. The figure is naturally born with left and right margins that would be nice to zero out. But we don't, exactly because we don't want to break that centering mechanism.

For that reason, I'd leave the margin change out from this PR, and let the theme handle that. Then, longer term, something like #29335 could potentially help address the fundamental issue.

Reply:
The margin rule has been removed from the front but not the editor.
In the editor, the margin only affects the list, not the actual block position, since the list is wrapped. 🤔

Reply:
Shouldn't the front and backend ideally be the same though? Or does the ol inherit from wp admin which diverges it anyway? 🙈

Reply:
Yes 🙊 , the list styles we are overriding are:
Browser defaults
common.css (wp admin)
and the [Editor Normalization Styles](https://github.com/WordPress/gutenberg/blob/trunk/packages/editor/src/editor-styles.scss)

Reply:
🙈 

I think it might still be best to remove that margin change, as it "paints over" an issue that we need to fix at the root, for example through bringing the iframe to the post editor, or #28878.

Reply:
🤷‍♀️ It will look different in the editor and front even without the wp admin styles.

Reply:
> 🤷‍♀️ It will look different in the editor and front even without the wp admin styles.

This is no different from what's currently in trunk, though, is it? I definitely agree it needs to be addressed, just want to make sure we do it right.